webplugin/css/app/mck-sidebox-1.0.css

```
@@ -4352,11 +4360,134 @@ ul.mck-intent-menu > li:hover {
     bottom: 0;
     left: 0;
     overflow: auto;
+    overflow: auto;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

overflow is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)